### PR TITLE
Bump version of tinygradient from 0.4.3 to 1.1.5

### LINKED ISSUE #4730: CVE-2019-11358 (Medium) detected in jquery-1.9.1.js
## CVE-2019-11358 - Medium Severity Vulnerability
<details><summary> Vulnerable Library - <b>jquery-1.9.1.js</b></p></summary>

<p>JavaScript library for DOM operations</p>
<p>Library home page: <a href="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.9.1/jquery.js">https://cdnjs.cloudflare.com/ajax/libs/jquery/1.9.1/jquery.js</a></p>
<p>Path to dependency file: /node_modules/tinygradient/bower_components/tinycolor/index.html</p>
<p>Path to vulnerable library: /node_modules/tinygradient/bower_components/tinycolor/demo/jquery-1.9.1.js</p>
<p>

Dependency Hierarchy:
  - :x: **jquery-1.9.1.js** (Vulnerable Library)
<p>Found in HEAD commit: <a href="https://github.com/opensearch-project/OpenSearch-Dashboards/commit/cba076465f44b6a819e3cff7986ff4cd21a66371">cba076465f44b6a819e3cff7986ff4cd21a66371</a></p>
<p>Found in base branch: <b>main</b></p>
</p>
</details>
<p></p>
<details><summary> Vulnerability Details</summary>
<p>  
  
jQuery before 3.4.0, as used in Drupal, Backdrop CMS, and other products, mishandles jQuery.extend(true, {}, ...) because of Object.prototype pollution. If an unsanitized source object contained an enumerable __proto__ property, it could extend the native Object.prototype.

<p>Publish Date: 2019-04-20
<p>URL: <a href=https://www.mend.io/vulnerability-database/CVE-2019-11358>CVE-2019-11358</a></p>
</p>
</details>
<p></p>
<details><summary> CVSS 3 Score Details (<b>6.1</b>)</summary>
<p>

Base Score Metrics:
- Exploitability Metrics:
  - Attack Vector: Network
  - Attack Complexity: Low
  - Privileges Required: None
  - User Interaction: Required
  - Scope: Changed
- Impact Metrics:
  - Confidentiality Impact: Low
  - Integrity Impact: Low
  - Availability Impact: None
</p>
For more information on CVSS3 Scores, click <a href="https://www.first.org/cvss/calculator/3.0">here</a>.
</p>
</details>
<p></p>
<details><summary> Suggested Fix</summary>
<p>

<p>Type: Upgrade version</p>
<p>Origin: <a href="https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-11358">https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-11358</a></p>
<p>Release Date: 2019-04-20</p>
<p>Fix Resolution: jquery - 3.4.0</p>

</p>
</details>
<p></p>

### PR DESCRIPTION
**Pull request by BSFishy**
### Description

Bump version of tinygradient from 0.4.3 to 1.1.5. The public API had no changes, and relevant tests pass.

### Issues Resolved

Fixes #4730

### Check List

- [x] All tests pass
  - [x] `yarn test:jest`
  - [x] `yarn test:jest_integration`
  - [x] `yarn test:ftr`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -217,7 +217,7 @@
     "source-map-support": "^0.5.19",
     "symbol-observable": "^1.2.0",
     "tar": "^6.1.11",
-    "tinygradient": "0.4.3",
+    "tinygradient": "1.1.5",
```
Comment (joshuarrrr) [inline package.json:220]: This is fine for `main`, but assuming `tinygradient` follows semver, this implies breaking changes. You noted no changes in the API, but can you summarize the `1.0.0` changes?

Comment (BSFishy) [reply package.json:220]: The main difference was a change in build system. It looks like they also fixed their typescript definition file, so it wouldn't seem like it was exporting functions that it wouldn't: https://github.com/mistic100/tinygradient/compare/0.4.3...1.0.0#diff-1c4c0cc198af965a5be46c21d5e65adf184698513a26f7c0ba54e8702a71de66

It doesn't seem like they were following semver to a T

Comment (AMoo-Miki) [inline package.json:220]: With v0 it made sense to have a fixed version but now that they have a non-pre release, we should use a pattern: `^1.1.5`

Comment (joshuarrrr) [reply package.json:220]: good call

**Review by AMoo-Miki — APPROVED** [review]
Thanks

**Review by joshuarrrr — APPROVED** [review]

**Pull request merged**